components/datetime/src/scaffold/calendar.rs

```diff
@@ -40,102 +40,153 @@ pub trait CldrCalendar: UnstableSealed {
 
     /// The data marker for loading skeleton patterns for this calendar.
     type SkeletaV1: DataMarker<DataStruct = PackedPatterns<'static>>;
+
+    /// Whether the given kind can be formatted with this impl's data
+    fn can_format_kind(kind: AnyCalendarKind) -> bool;
```

Comment:
thought: we used to have such a function

Reply:
The function is flipped: it is implemented on `Date<A>` and it takes an `AnyCalendarKind`. But, I can't get the `AnyCalendarKind` from a `FixedCalendarDateTimeFormatter`.